device-types/Eaton/Tripp-Lite-SMART500RT1U.yaml

```diff
@@ -0,0 +1,24 @@
+---
+manufacturer: Eaton
+model: Tripp Lite SMART500RT1U
+slug: eaton-tripp-lite-smart500rt1u
+u_height: 1
+is_full_depth: false
+front_image: true
+rear_image: true
```

Comment:
Datasheet link for comments `https://assets.tripplite.com/product-pdfs/en/smart500rt1u.pdf`